docs/development.rst

```diff
@@ -197,7 +197,9 @@ To include the changes that the hooks made, ``git add`` any
 files that were modified and run ``git commit`` (or, use ``git commit -a``
 to commit all changed files.)
 
-If you would like to run the checks without committing, use ``pre-commit run``.
+If you would like to run the checks without committing, use ``pre-commit run``
```

Comment:
Maybe we should suggest `pre-commit run --all` then the caveat is unneeded?

Reply:
good idea